### PR TITLE
Update GettingStarted.md

### PR DESCRIPTION
**Pull request by JeremyTHolland**
Added a couple more current choices for MUD clients since zMUD development seems to have stopped. Last release was almost 10 years ago if I'm reading the download page right.

### CODE REVIEW

**Comment by JeremyTHolland** [comment]
Oops. I didn't realize I was reading the docs on the main repo before I made the some edits and submitted a PR. I can resubmit it from my fork and PR from there back to here if you want. Its been a busy weekend and I'd like to blame tiredness for my lack of attention to detail. 😄

File: Documentation/GettingStarted.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ Satisfy the [Basic Prerequisites](BasicPrerequisites.md) if you haven't already.
 * Ensure you have a Telnet client available, such as:
   - Telnet.exe: Built into all supported versions of Windows, but the Telnet Client feature generally starts disabled. Open up the "Turn Windows features on and off" control panel, scroll down and ensure "Telnet Client" gets selected. To see if you have one ready, open up a Command Prompt and run "telnet localhost 4000", which should print that it's connecting to localhost, but then times out (assuming your WheelMUD server isn't running yet).
   - [zMUD or cMUD](http://www.zuggsoft.com/index.php). Heavy, feature-rich telnet client, used by tons of serious MUD users.
+  - [Mudlet](https://www.mudlet.org/) Full featured like zMUD, but free and open source ([code](https://github.com/Mudlet/Mudlet)).
```
Comment (DavidRieman) [inline Documentation/GettingStarted.md:10]: nit: Should have period after `[Mudlet](https://www.mudlet.org/).` for consistency.
nit: Ditto for next line.

**Review by DavidRieman — APPROVED** [review]
Looks good to me.  Submitting here was reasonable, especially so for small doc updates.

**Review by JeremyTHolland — COMMENTED** [review]
I forgot the periods. Period.

**Comment by JeremyTHolland** [comment]
I think I got confused here. Was I supposed to do something to close this?

**Comment by DavidRieman** [comment]
For GitHub process, normally what you would do would be push a new commit to your branch (JeremyTHolland:patch-1) however since that format suggests you probably used the inline markdown editor to make the branch...  It's a little awkward and funny enough I haven't run into this situation yet.  Your options look to me like:
* Just merge it and open another with the mods via the inline editor.
* Pretend this was code and practice the code process for it:  git checkout the `JeremyTHolland:patch-1` branch locally, edit the .md file on your computer with the additional change, git add, git commit it, git push it.  That'll make this PR see the new commit and be ready for re-review in the way you'd do it with code.

**Review by DavidRieman — APPROVED** [review]
Perfect!  Can you tell me if the Merge button is green for you?  Err, after AppVeyor passes, that is...  I don't know right now if GitHub expects me to do it or if approval gives you the option.

**Comment by JeremyTHolland** [comment]
Ok got it this time. I got confused because the documentation files don't show in VS in the solution even thought they're in the repo. VS Code to the rescue! Thanks for the explanation on process.

**Comment by DavidRieman** [comment]
Aha!  Actually yeah I have been meaning to add a folder link to the documentation in the solution for easy discovery even inside Visual Studio.  I'll look into that.

**Pull request merged**